src/IPHlpApi/IPHlpApi.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+#if NETFRAMEWORK || NETSTANDARD2_0_ORLATER
+    using System.Net.Sockets;
+    using System.Runtime.ConstrainedExecution;
+#endif
+    using System.Runtime.InteropServices;
+    using System.Security;
+
+#if NETSTANDARD1_1
+    using AddressFamily = System.Int32;
```

Comment:
I'm not sure using `AddressFamily` here as defined by .NET is ever appropriate. It apparently defines a bunch of enum values while the documentation only claims to return two of them. Do the ordinal values of each of the .NET enum values equal the ones from native code, at least?

Reply:
`AddressFamily` maps directly to the [various `AF_...` values](https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-socket) used by the Win32 API. `AddressFamily.InterNetwork = 2 = AF_INET` and `AddressFamily.InterNetworkV6 = 23 = AF_INET6`